app/config_default.ini

```diff
@@ -1,11 +1,11 @@
 [general]
 # web site url = eg. 'http://' + domain_name + '/upload'
-domain_name = review.px4.io
+domain_name = domain_name
 # http or https
 http_protocol = http
 
 # path for everything that will be stored on disk (including the database)
-storage_path = data
+storage_path = /opt/service/data
```

Comment:
I changed this because it seems like the plot_app/config check the storage_path whether is absolute, if not it just use the ../data instead. Within a container i think use a aboslute path is better. Or maybe we just change the config.py to use ../../data if the storage_path is not absolute?